### ISSUE TITLE
Resnet31, maxpooling (1,2) or (2,1)

### ISSUE THREAD

**Issue by yongduek** [wangpengnorman/SAR-Strong-Baseline-for-Text-Recognition#4]
Many thanks for sharing the source code. I am trying to implement this with a DNN package other than torch7. 

What is the meaning of `Max(1,2,1,2)`. Based on the paper saying "we also use a 1 × 2 max-pooling layer as in (Shi, Bai, and Yao 2017), which reserves more information along the horizontal axis", it seems to mean that the mask window for max-pooling should be `(width,height) = (1,2)`. Is this right?

**Comment by Pay20Y** [comment]
I agree with you. Refer to the source code the final feature map's size is (6,40).
